pkgs/development/libraries/graphene-hardened-malloc/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "graphene-hardened-malloc-${version}";
+  version = "PQ2A.190405.003.2019.04.01.19";
```

Comment:
This version string will fail to parse, as in

> nix-repl> builtins.parseDrvName "graphene-hardened-malloc-PQ2A.190405.003.2019.04.01.19"
{ name = "graphene-hardened-malloc-PQ2A.190405.003.2019.04.01.19"; version = ""; }

Reply:
Damn. Well, seeing as it seems to be a thinly disguised datestamp, I could reduce it to `190405.003.2019.04.01.19`?

Reply:
Generally, `version` should at least satisfy `newer > older`, beyond that I guess whatever is most convenient to maintain.